fairscale/nn/data_parallel/sharded_ddp.py

```diff
@@ -102,7 +96,7 @@ def __init__(
         process_group: Any = None,
         broadcast_buffers: bool = True,
         sync_models_at_startup: bool = True,
-        reduce_buffer_size: int = 0,
+        reduce_buffer_size: int = 2 ** 23,
```

Comment:
nit: a comment about why we are using this as the default size would be nice.

Reply:
there's a word about it in the argument docstring, let me know what you think ?

Reply:
Should the doc string be updated? i think it still says 0.

I just meant to ask why we think this is a good number for the reduce buffer. (I thought it would be useful for the user to know as well. It's a nit, so feel free to ignore.).